recipes/rkp/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set name = "RKP" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: e7cda9bae969b6f1a22cf7a96e547bd488e649f1969b8a398a344704302f58d0
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: generic
+
+requirements:
+  host:
+    - argparse >=1.4.0
+    - biopython >=1.76
+    - matplotlib >=3.1.2
+    - numpy >=1.17.3
+    - pandas >=0.25.3
+    - pip
+    - python >=3.6
+    - tqdm >=4.41.1
+    - kmc >=3.1.1
+    - bowtie2 >=2.3.5
+    - bedtools >=2.29.2
+    - r >=3.6
+    - r-pheatmap >=1.0.12
+    - r-gplots >=3.0.1.1
+    - blast >=2.9.0
+    - samtools >=1.10
+    - bedops >=2.4.37
+    - seqkit >=0.11.0
+  run:
```

Comment:
Please check which packages are required at host and runtime (https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#export-runtime-requirements)